src/ubx.cpp

```diff
@@ -108,10 +108,12 @@ GPSDriverUBX::configure(unsigned &baudrate, OutputMode output_mode)
 {
 	_configured = false;
 	_output_mode = output_mode;
+#if !defined(__PX4_POSIX_RPI)
```

Comment:
I think it would be better if we add the interface information to the GPSDriverUBX constructor since the interface information is static?